src/components/spatialdisplay/SpatialDisplay.vue

```diff
@@ -41,15 +58,83 @@ const route = useRoute()
 const router = useRouter()
 const { mobile } = useDisplay()
 
+const settings = useUserSettingsStore()
+const baseUrl = configManager.get('VITE_FEWS_WEBSERVICES_URL')
+const { layerCapabilities, times } = useWmsLayerCapabilities(
+  baseUrl,
+  () => props.layerName,
+)
+
+function getFilterActionsFilter(): filterActionsFilter {
+  return {
+    locationIds: props.locationId,
+    filterId: props.filterIds ? props.filterIds[0] : undefined,
+  }
+}
+
+function getTimeSeriesGridActionsFilter():
+  | timeSeriesGridActionsFilter
+  | undefined {
+  if (!props.longitude || !props.latitude) return
+  if (!layerCapabilities.value?.boundingBox) return
+  if (!layerCapabilities.value?.firstValueTime) return
+  if (!layerCapabilities.value?.lastValueTime) return
+
+  const [x, y] = toMercator([props.longitude, props.latitude])
+
+  const { minx, miny, maxx, maxy } = layerCapabilities.value.boundingBox
+  const bbox = [minx, miny, maxx, maxy].map(Number)
+
+  const startTime = layerCapabilities.value.firstValueTime
+  const endTime = layerCapabilities.value.lastValueTime
+
+  return {
+    layers: props.layerName,
+    x,
+    y,
+    startTime,
+    endTime,
+    bbox,
+    documentFormat: 'PI_JSON',
+    elevation: elevation.value ?? layerCapabilities.value.elevation?.lowerValue,
+  }
+}
+
+const options = computed<UseDisplayConfigOptions>(() => {
```

Comment:
`options` is too generic. Rename to `displayConfigOptions` or to `filterOptions`.